tft.py

```diff
@@ -116,30 +116,68 @@ def kill_process(process_executable: str, force: bool = True) -> subprocess.Comp
     )
 
 
+def launch_league_client(deceive_config: bool) -> None:
+    """
+    Launches the client
+
+    Args:
+        deceive_config: Whether to open the client through Riot or through Deceive
+    """
+    if deceive_config:
+        location = config.get_install_location_deceive()
+        if location is None:
+            logger.warning("Deceive is enabled in config.yaml, but no path was given! The bot will attempt to find the .exe on its own")    # pylint: disable=line-too-long
+            location = system_helpers.determine_deceive_install_location()
+            if location is None:
+                logger.error("Could not determine Deceive location. Please manually add the location in output/config.yaml")    # pylint: disable=line-too-long
+                logger.warning("Bot will now continue without Deceive")
+                config.update_deceive_config(update=False)
+                launch_league_client(deceive_config=False)
+                return
+
+            logger.info(f"Found Deceive at: {location}")
+        else:
+            logger.debug(f"Using deceive with the given path: {location}")
+
+        subprocess.Popen(   # pylint: disable=consider-using-with
+            location,
+            stdin=None,
+            stdout=None,
+            stderr=None,
+            close_fds=True,
+            creationflags=DETACHED_PROCESS,
+        )

```

Comment:
This is a bit of a nit, but I'd say since lines 142-149 & 156-162 are identical, it would be good to extract these to assign a common `executable_args_to_launch` variable in both branches of the `if deceive_config:` (L126) check, and then launch the process using `args=executable_args_to_launch`, or something along those lines.

Thoughts?

Reply:
Sounds good, I didn't think about that while writing it.